website/docs/reference/supported_commands.md

```diff
@@ -639,3 +639,54 @@ db.aggregate()
+
+## Diagnostic commands
+
+| Command           | Argument                | Status | Comments                                                  |
+|-------------------|-------------------------|--------|-----------------------------------------------------------|
+| `buildInfo`       |                         | ✅     | Basic command is fully supported                          |
+| `collStats`       |                         | ✅     | Basic command is fully supported                          |
+|                   | `collStats`             | ✅     |                                                           |
+|                   | `scale`                 | ✅     |                                                           |
+| `connPoolStats`   |                         | ❌     | Unimplemented                                             |
+| `connectionStatus`|                         | ✅     | Basic command is fully supported                          |
+|                   | `showPrivileges`        | ✅     |                                                           |
+| `dataSize`        |                         | ✅     | Basic command is fully supported                           |
+|                   | `keyPattern`            | ⚠️     | Unimplemented                                             |
+|                   | `min`                   | ⚠️     | Unimplemented                                             |
+|                   | `max`                   | ⚠️     | Unimplemented                                             |
+|                   | `estimate`              | ⚠️     | Ignored                                                   |
+| `dbHash`          |                         | ❌     | Unimplemented                                             |
+|                   | `collection`            | ⚠️      |                                                           |
+| `dbStats`         |                         | ✅     | Basic command is fully supported                          |
+|                   | `scale`                 | ✅     |                                                           |
+|                   | `freeStorage`           | ⚠️      | Unimplemented                                             |
+| `driverOIDTest`   |                         | ⚠️      | Unimplemented                                             |
+| `explain`         |                         | ✅     | Basic command is fully supported                          |
+|                   | `verbosity`             | ⚠️      | Ignored                                                   |
+|                   | `comment`               | ⚠️      | Unimplemented                                             |
+| `features`        |                         | ❌     | Unimplemented                                             |
```

Comment:
This command is stated as internal. Do we plan to implement such commands?

Reply:
@AlekSi PTAL

Reply:
No, but we will remove them at future passes